### DISCUSSION TITLE
Popup print preview in dialog.ts

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6770] [Help]
Please after clicking `.btn-save-and-new` , is there a way to popup print preview automatically on the same dialog page like the below, or if not possible then popup on a new tab
I already have the preview button on the Grid.ts, but I dont want to go back to the main list before clicking the print preview

![image](https://github.com/serenity-is/Serenity/assets/50826372/75c79b86-cd76-4524-aef3-e16c8dc8859b)

**Comment by flashyjunior** [comment]
Please any help will be greatly appreciated
